content/en/docs/Tracing/tempo-walkthrough.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Tracing with Grafana Tempo
+description: Tracing with Grafana Tempo.
+weight: 10
+---
+
+## Cluster Setup
```

Comment:
"This tutorial walks through a proof-of-concept setup of a development Kubernetes cluster and the components needed to demonstrate distributed tracing in Kyverno. This tutorial assumes the use of KinD and a system where ports 80 and 443 have not already been bound to other services.

Reply:
Done